_data/jobs.yml

```diff
@@ -1,3 +1,13 @@
+- expires: 2022-02-01
```

Comment:
Just to check - you only want them up for like a week?

Reply:
Well, that's a good question.  It *is* what the submitter put it the form.

Reply:
okay, then that's what we'll do!

Reply:
Confirmed the expiration date.  All good!